tailwind.config.js

```diff
@@ -1,37 +1,52 @@
 const colors = require("tailwindcss/colors");
+const defaultTheme = require("tailwindcss/defaultTheme");
 
 module.exports = {
   mode: "jit",
   purge: ["./pages/**/*.{js,ts,jsx,tsx}", "./components/**/*.{js,ts,jsx,tsx}"],
   darkMode: false, // or 'media' or 'class'
   theme: {
     extend: {
+      fontFamily: {
+        heading: ['"Baloo 2"', ...defaultTheme.fontFamily.sans],
+        sans: ['"Helvetica"', ...defaultTheme.fontFamily.sans],
```

Comment:
did you need to add that main.css - wouldn't just changes in this config file do the job?

Reply:
Yeh i did in order to change the content in the notion render.